userguide/dailyuse/anbox.rst

```diff
@@ -68,12 +67,12 @@ How to install new apks
 Keep your apps up to date
 -------------------------
 
-- To keep your apps running the lasted version you can make use of F-Droid or ApkTrack. If you like to install one of the above apps you can find them here:
+- To keep your apps up to date you can use of F-Droid or ApkTrack. If you want to install any of the above apps you can find them here:
 
 - F-Droid: https://f-droid.org/
 - ApkTrack: https://f-droid.org/packages/fr.kwiatkowski.ApkTrack/
 
-How to uninstall applications
+How to uninstall apps
 -----------------------------
```

Comment:
When changing the length of a title, the length of its underline should also be changed